filters/builtin/compress_test.go

```diff
@@ -373,6 +387,42 @@ func TestCompress(t *testing.T) {
 		http.Header{
 			"Content-Encoding": []string{"deflate"},
 			"Vary":             []string{"Accept-Encoding"}},
+	}, {
+		"brotli",
```

Comment:
I think we do not test custom encoding preference order.
Maybe we could add `encodings []string` to the test case struct and when it is not empty then redefine compress filter using `NewCompressWithOptions` like we do in skipper.go.

This way we can test that e.g. `-compress-encodings=br,gzip` indeed picks `br` over `gzip` when `Accept-Encoding` has the Chrome/Firefox default value of `gzip, deflate, br`.

Reply:
good catch, added unit test